editor/components/block-list/style.scss

```diff
@@ -751,51 +765,48 @@
 	pointer-events: none;
 	height: $block-toolbar-height;
 
-	// Position toolbar below the block on mobile
+	// Position toolbar below the block on mobile.
 	position: absolute;
-	bottom: -$block-toolbar-height + 1px;
+	bottom: $block-toolbar-height;
 	left: $block-padding;
 	right: $block-padding;
-	box-shadow: $shadow-below-only;
 
-	// Position the contextual toolbar above the block, add 1px to each to stack borders
+	// Position the contextual toolbar above the block.
 	@include break-mobile() {
 		position: sticky;
 		bottom: auto;
 		left: auto;
 		right: auto;
-		box-shadow: none;
-		margin-top: -$block-toolbar-height - 1px;
-		margin-bottom: $block-padding + 1px;
+		margin-top: -$block-toolbar-height - $border-width;
+		margin-bottom: $block-padding + $border-width;
 	}
 
 	@include break-small() {
-		top: -1px;	// stack borders
+		top: -$border-width;
 	}
 
-	// Floated items have special needs for the contextual toolbar position
+	// Floated items have special needs for the contextual toolbar position.
 	.editor-block-list__block[data-align="left"] &,
 	.editor-block-list__block[data-align="right"] & {
-		margin-bottom: 1px;
-		margin-top: -$block-toolbar-height - 1px;
+		margin-bottom: $border-width;
+		margin-top: -$block-toolbar-height - $border-width;
 	}
 
-	// put toolbar snugly to side edges on mobile
-	margin-left: -$block-padding - 1px;	// stack borders
-	margin-right: -$block-padding - 1px;
+	// Put toolbar snugly to side edges on mobile.
```

Comment:
"Snugly" is a cute turn of phrase but is a bit of a colloquialism. Maybe there's a better way to phrase this for less adapt English folk?